src/blog/0.2/version-0.2-released.vmd

```diff
@@ -0,0 +1,443 @@
+---
+title: Vale 0.2 Released: Higher RAII, Concept Functions, Const Generics, FFI, Modules, Faster Compiles, set Keyword
+author: Evan Ovadia
+date: May 10, 2022
+realm: blog
+path: blog/version-0.2-released
+layout: annotated
+namespace: c-blog m-annotated
+---
+
+
+[Version 0.2](https://vale.dev/download) is now out of beta, and officially released!
+
+
+As we all know, Vale is a programming language that aims to be *fast*, *safe*, and *easy to use.* It's a challenging goal, and version 0.2 lays solid foundations for the next step in the endeavor.
+
+
+This version has been prioritizing ease of use, making the compiler usable for larger projects. We can now comfortably create large programs (such as [this roguelike game](https://verdagon.dev/blog/higher-raii-7drl)). With this version, Vale has graduated from a prototype to early alpha status.
+
+
+You can find the [version 0.2 binaries here](https://github.com/ValeLang/Vale/releases/tag/v0.2.0.0).
+
+
+Try it out, and if you like what you see, please consider [sponsoring us on GitHub!](https://github.com/sponsors/ValeLang) This is a free project for the good of all, and with your help, we can continue working on it for longer. [#longer]
+
+
+Below are the new features in 0.2, enjoy!
+
+
+# Higher RAII
+
+This release enables [Higher RAII](http://localhost:8000/guide/structs#higher-raii), a form of linear typing that helps the compiler enforce that we "never forget to call that function".
+
+
+For a real-world use case, check out the recent article [Higher RAII, the pattern that saved me a vital 5 hours in the 7DRL Challenge](https://verdagon.dev/blog/higher-raii-7drl).
+
+
+<<<<
+To use Higher RAII, just add a `#!DeriveStructDrop` above your struct or `#!DeriveInterfaceDrop` above your interface, to prevent it from automatically defining a `drop` function.
```

Comment:
`#!DeriveStructDrop` means don't derive drop right? I'm guessing the `!` means 'don't' in this case. I would be clear about this because I initially thought the annotation was named the opposite of what it does.